run_hyperion.sh

```diff
@@ -124,20 +128,37 @@ if [[ $START == 1 ]]; then
     source .venv/bin/activate
 
     #Add future arguments here
-    declare -A args=( ["IN_DEV"]="$IN_DEV" ["SKIP_STARTUP_CONNECTION"]="$SKIP_STARTUP_CONNECTION" ["VERBOSE_EVENT_LOGGING"]="$VERBOSE_EVENT_LOGGING"
-                    ["LOGGING_LEVEL"]="$LOGGING_LEVEL")
-    declare -A arg_strings=( ["IN_DEV"]="--dev" ["SKIP_STARTUP_CONNECTION"]="--skip-startup-connection" ["VERBOSE_EVENT_LOGGING"]="--verbose-event-logging"
-                            ["LOGGING_LEVEL"]="--logging-level=$LOGGING_LEVEL")
-
-    commands=()
-    for i in "${!args[@]}"
+    declare -A h_only_args=(        ["SKIP_STARTUP_CONNECTION"]="$SKIP_STARTUP_CONNECTION"
```

Comment:
Could: Having just `h` here (and in the file as a whole) is not so obvious as to what you mean. Maybe `hyperion_main_only_args`?